sandbox-backend/src/main/java/com/exadel/sandbox/team5/service/impl/OrderServiceImpl.java

```diff
@@ -101,4 +103,21 @@ public Order createOrder(Long discountId, int maxOrderSize, long amountDiscountD
     private List<Order> activeOrdersByTime(List<Order> activeOrders) {
         return activeOrders.stream().filter(e -> System.currentTimeMillis() < e.getPromoCodePeriodEnd().getTime()).collect(Collectors.toList());
     }
+
+    @Override
+    public List<List<OrderDto>> getOrdersByDiscountIds(List<Long> discountIds) {
```

Comment:
Will it be usable? How UI will know about company ids? What a reason to return full orders list for statistic? Just total count should be enough

Reply:
sounds logic, agree)